cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,32 @@
+class IframePage {
+
+    private w3schoolSiteUrl: string;
+    private defaulIframe: string;
+    private cssH1: string;
+    private tutorialSpan: string;
+
+    constructor() {
+        this.w3schoolSiteUrl = 'https://www.w3schools.com/html/html_iframe.asp';
+        this.defaulIframe = '[src="default.asp"]';
+        this.cssH1 = '#main > h1';
```

Comment:
if this selector is not used, then remove it

Reply:
Done!